### DISCUSSION TITLE
Mario Artist: Talent Studio graphical glitch

### DISCUSSION THREAD

**Discussion by MarineDog** [mupen64plus-ae/mupen64plus-ae#952] [General]
![Screenshot_20210903-202504_(1)](https://user-images.githubusercontent.com/90079296/132080976-5b5d3074-9d07-4d0a-b53c-2f3a7b16836f.png)
As can be seen (hopefully), there is a graphical glitch in Mario Artist: Talent Studio where faces, after attempts at modification, become some manner of garbage texture. I have attempted to fix this by changing my emulation profile, but all of the defaults retain this bug.

This is on an LG G8, running Android 10.

**Comment by fzurita** [comment]
Try using the software renderer emulation profile. That shouldn't have that bug.

**Reply by MarineDog** [reply]
That was the one I first tried.

**Reply by fzurita** [reply]
Ok, then it's probably not a graphics plugin issue. Can you create a custom emulation profile and change the R4300 emulator to cached interpreter and see if that helps?

The game will probably run a lot slower, but just to help understand where the issue is.

**Reply by MarineDog** [reply]
I am unsure how exactly to do that.

**Reply by fzurita** [reply]
Go to profiles then emulation, select Glide64-Accurate and hit copy. Give it a name and hit ok. You will then be able to change parameters for that profile. In there, change the R4300 emulator to cached interpreter.

Once done, go back to the game list. Select the game, go to settings, change the emulation profile to the one you created.

**Reply by MarineDog** [reply]
Okay, thank you! I'll let you know how it goes!

**Reply by MarineDog** [reply]
Okay, I realize I think I may have gotten it working. I have found that out of the two versions where I acquired Talent Studio, the version actually meant for emulators does not function at all, but the version meant for flashcarts, functions. I tried the modified Glide64-Accurate emulation profile copy and visuals were not existent. Then I tried the. Software Renderer and it works, albeit slower than my previous go around. The face glitch seems to be fixed at first glance, although I'll be doing more testing to see if stuff has been actually solved and the like.
